pubspec.yaml

```diff
@@ -4,21 +4,22 @@ version: 3.0.0
 homepage: https://docs.atsign.com/
 
 environment:
-  sdk: '>=2.16.1 <3.0.0'
+  sdk: '>=3.0.0 <4.0.0'
 
 
 dependencies: 
```

Comment:
As this app is using packages rather than publishing a package it would be good to get the dependencies pinned down.

Reply:
Lets chat about a company wide strategy for this